### PR TITLE
Request a PDF from the PDF API

### PR DESCRIPTION
**Pull request by emileswarts**
This PDF will be unused after generated, next PR will use it as an attachment.

### CODE REVIEW

File: app/services/process_submission_service.rb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+# rubocop:disable all
```
Comment (emileswarts) [inline app/services/process_submission_service.rb]: I've crossed a threshold of loc and complexity.
This class needs a refactor, don't want to do it as a part of this feature.

File: app/value_objects/pdf_submission_detail.rb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+class PdfSubmissionDetail
+  def initialize(params = {}); end
+end
```
Comment (emileswarts) [inline app/value_objects/pdf_submission_detail.rb]: noop, needs to be refactored out

Comment (sequencemedialimited) [reply app/value_objects/pdf_submission_detail.rb]: When? :)

File: spec/requests/submission_spec.rb [hunk 3]
```diff
@@ -12,6 +12,10 @@
   # ignores it in an initializer
   let(:user_identifier) { SecureRandom.uuid }
 
+  before do
+    ENV['PDF_GENERATOR_ROOT_URL'] = 'http://pdf-generator.com'
```
Comment (sequencemedialimited) [inline spec/requests/submission_spec.rb]: What's this for?

Comment (sequencemedialimited) [reply spec/requests/submission_spec.rb]: Argh! It's a spec, ignore me

File: app/value_objects/pdf_submission_detail.rb (hunk 2 above)
Comment (emileswarts) [reply app/value_objects/pdf_submission_detail.rb]: Would prefer to tackle the process_submission_service as an isolated task.
Will push for this before we modify this area of the code again.

**Pull request merged**